pkgs/applications/misc/polybar/default.nix

```diff
@@ -39,6 +39,13 @@ stdenv.mkDerivation rec {
       maintainers = [ maintainers.afldcr ];
       platforms = platforms.unix;
     };
+    # This patch should be removed with next stable release.
+    patches = [
+      (fetchurl {
```

Comment:
You should use `fetchpatch` here, which will cannibalize the patch so the hash doesn't change later (github likes to change the contents subtly, despite the semantics of the patch remaining unchanged).

Reply:
done.